### PR TITLE
[production/RRFS.v1] Remove all references to graphics and archive jobs from RRFSv1

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 

- In PR #331 , the graphics and archive J-jobs and ex-scripts were removed. In this PR, those tasks are removed from FV3LAM_wflow.xml, all related variables are removed from files in the ush directory, and python_graphics is removed as an option from Externals.cfg. I was able to build the DA workflow successfully with these changes so I don't think I broke anything.
- This PR does not contain the changes from PR #332 .  So there may be conflicts whenever one PR is merged before the other.

## TESTS CONDUCTED: 

Did not test any jobs but successfully built the workflow.

### Machines/Platforms:

- WCOSS2
  - [x] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [x] Non-DA engineering test
  - [x] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [ ] Parallel
- [ ] RRFS fire weather
- [ ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

### CODE REVIEW

File: ush/setup.sh
Comment (MatthewPyle-NOAA) [inline ush/setup.sh:1]: Are the NCARG and NCL items needed?  Would assume they are plotting related.

**Review by MatthewPyle-NOAA — COMMENTED** [review]
One question, but mostly looks good to me.

File: ush/setup.sh
Comment (BenjaminBlake-NOAA) [reply ush/setup.sh:1]: I was wondering that too, and I think they are.  I'll go ahead and remove them.

**Review by MatthewPyle-NOAA — APPROVED** [review]
Looks good - merging

**Pull request merged**